scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -90,7 +90,7 @@ def cice_hist2fms(input_filename, output_filename):
     ds = xr.open_dataset(input_filename)
 
     if 'aicen' in ds.variables and 'hicen' in ds.variables and 'hsnon' in ds.variables:
-        logging.info(f"*** Already reformatted, skipping.")
+        print(f"*** Already reformatted, skipping.")
```

Comment:
why are all these logging replaced with print?

Reply:
see PR description @CoryMartin-NOAA .

Reply:
Ok but why? as it presumably was working before? I'd prefer to find the reason.

Reply:
@CoryMartin-NOAA IIRC there was speculation that it was something coming down from global-workflow - we haven't figured it out and it's easy to change back to logging when we do

Reply:
I think we all agree on that @CoryMartin-NOAA ! @AndrewEichmann-NOAA , can you revert your `prints` back to `loggings` and submit an issue? I thought we had an issue describing this problem already, but I can't find it.